### PR TITLE
ClientDetailContainer Jump card

### PR DESCRIPTION
**Pull request by aringocode**
### Before:

https://user-images.githubusercontent.com/65866093/203057366-bce80a05-8b0a-40c0-8f5e-0d680bfcc91f.mp4

### After:
For minimally filled-in information, the card doesn't jump. If more `Redirect URIs` is added, the card jumps and starts jumps when saving and editing.

https://user-images.githubusercontent.com/65866093/203065431-7f972cfc-2016-4d5c-a343-23662962f208.mp4

#### Changes on small screen: 

- On small screens I can remove the height for the `Redirect URIs` or leave it as in the video

https://user-images.githubusercontent.com/65866093/203066298-ea849d04-1611-47e7-9cb1-633b29492c7f.mp4

### CODE REVIEW

**Comment by aringocode** [comment]
@Pe5h4 Can you review pls, and say what you think of these changes I showed in the video. What you think is the best way to do it?

**Comment by aringocode** [comment]
@Pe5h4 I added new changes
Can you review it pls?

https://user-images.githubusercontent.com/65866093/203505424-b7d2faf8-1b65-4c71-ab5a-152e2bb9c824.mp4

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
@Aringoaway When I see the code, it looks to me that this change has increased the complexity quite a lot and so far it does not seem to me like a appropriate structural change (sorry, my bad, I was also not sure how to do it properly, so I guess I mislead you).

I still have a feeling, that there is something not correct in the current code in main branch (the state/request/response handling) and thus it causes displaying of old values for a moment. We can have a sync about it.

**Comment by aringocode** [comment]
@Pe5h4 Can you review pls?

https://user-images.githubusercontent.com/65866093/203818805-32e09269-9f49-4e12-953d-cf70c2f98267.mp4

File: src/modules/auth/clients/ClientDetailContainer.js [hunk 1]
```diff
@@ -116,15 +118,24 @@ const ClientDetailContainer = (props) =>  {
 			if (response.statusText != 'OK') {
 				throw new Error("Unable to change client details");
 			}
-			props.app.addAlert("success", t("ClientDetailContainer|Client updated successfully"));
+			setClient((prevState) => {
```
Comment (Pe5h4) [inline src/modules/auth/clients/ClientDetailContainer.js]: @Aringoaway I think you can do just:

```
setClient({...client, redirect_uris: uri})
```

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
Ok, this looks good, only minor remark

**Comment by aringocode** [comment]
@Pe5h4 Can you review again pls?

File: src/modules/auth/clients/ClientDetailContainer.js [hunk 2]
```diff
@@ -108,6 +108,8 @@ const ClientDetailContainer = (props) =>  {
 			body.client_name = "";
 		}
 
+		setDisabled(true);
```
Comment (Pe5h4) [inline src/modules/auth/clients/ClientDetailContainer.js]: @Aringoaway Put `setDisabled(true)` between lines 92 and 93 (before `body` declaration)

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
Minor change

**Review by Pe5h4 — APPROVED** [review]

**Pull request merged**